data/org.gnome.shell.gschema.xml.in

```diff
@@ -144,6 +144,14 @@
       <summary>Keybinding that pauses and resumes all running tweens, for debugging purposes</summary>
       <description></description>
     </key>
+
```

Comment:
I reviewed the changes after the initial commit, but I don't see anything worth mentioning in the commit log, so I'll rephrase the current message to at least not to contain obsolete information.